<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>860</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>曹氏任务管理器</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gird2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="grid">
      <item row="0" column="0">
       <widget class="QTabWidget" name="tabWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>Noto Serif CJK SC</family>
          <pointsize>13</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <property name="elideMode">
         <enum>Qt::ElideNone</enum>
        </property>
        <widget class="QWidget" name="tab">
         <attribute name="title">
          <string>系统信息</string>
         </attribute>
         <widget class="QLabel" name="_lbDevName">
          <property name="geometry">
           <rect>
            <x>40</x>
            <y>40</y>
            <width>81</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string>设备名称</string>
          </property>
         </widget>
         <widget class="QLabel" name="_lbStartTime">
          <property name="geometry">
           <rect>
            <x>40</x>
            <y>80</y>
            <width>81</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string>启动时间</string>
          </property>
         </widget>
         <widget class="QLabel" name="_lbRunTime">
          <property name="geometry">
           <rect>
            <x>40</x>
            <y>120</y>
            <width>81</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string>运行时间</string>
          </property>
         </widget>
         <widget class="QLabel" name="_lbSysVersion">
          <property name="geometry">
           <rect>
            <x>40</x>
            <y>160</y>
            <width>81</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string>系统版本</string>
          </property>
         </widget>
         <widget class="QLabel" name="_lbCpuInfo">
          <property name="geometry">
           <rect>
            <x>40</x>
            <y>210</y>
            <width>81</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string>CPU信息</string>
          </property>
         </widget>
         <widget class="QTextBrowser" name="tbrCpuInfo">
          <property name="geometry">
           <rect>
            <x>185</x>
            <y>211</y>
            <width>571</width>
            <height>261</height>
           </rect>
          </property>
         </widget>
         <widget class="QLabel" name="lbDevName">
          <property name="geometry">
           <rect>
            <x>190</x>
            <y>40</y>
            <width>391</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
         <widget class="QLabel" name="lbStartTime">
          <property name="geometry">
           <rect>
            <x>190</x>
            <y>80</y>
            <width>391</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
         <widget class="QLabel" name="lbRunTime">
          <property name="geometry">
           <rect>
            <x>190</x>
            <y>120</y>
            <width>391</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
         <widget class="QLabel" name="lbSysVersion">
          <property name="geometry">
           <rect>
            <x>190</x>
            <y>160</y>
            <width>391</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="tab_3">
         <attribute name="title">
          <string>进程</string>
         </attribute>
        </widget>
        <widget class="QWidget" name="tab_2">
         <attribute name="title">
          <string>服务</string>
         </attribute>
        </widget>
        <widget class="QWidget" name="tab_4">
         <attribute name="title">
          <string>性能</string>
         </attribute>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="statusTip">
    <string/>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
